build.gradle

```diff
@@ -63,7 +63,7 @@ dependencies {
     compile 'com.intellij:annotations:+@jar'
 }
 
-group = "com.structurize"
+group = "com.ldtteam.structurize"
```

Comment:
Not sure if "structurize" remains in the group string (I have it in the second pr)

Reply:
This is not correct
The group is com.ldtteam
The artifactbasename is structurize,